SU2_CFD/include/output/CElasticityOutput.hpp

```diff
@@ -37,6 +37,7 @@
 class CElasticityOutput final: public COutput {
 protected:
 
+  const CSolver *heat_solver = nullptr; //!< Pointer to the heat solver
```

Comment:
This doesn't need to be a member variable, the output function have access to the solver container, see CElasticityOutput::LoadHistoryData